src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -703,5 +703,15 @@ private void OptionsButton_Click(object sender, RoutedEventArgs e)
             grid.ContextMenu.DataContext = viewModel;
             grid.ContextMenu.IsOpen = true;
         }
+
+        private void UnDismissMessage_Click(object sender, MouseButtonEventArgs e)
+        {
+            if (!(sender is Grid grid)) return;
+
+            TextBlock textBlock = WpfUtilities.ChildOfType<TextBlock>(grid);
+            if (textBlock == null) return;
+
+            viewModel.ErrorBubble.UndismissMessageCommand.Execute(textBlock.Tag);
```

Comment:
Can you put some comments? I think it is specifically dismissing particular alerts?

Reply:
"Fires when the user clicks on the node context menu's list of dismissed messages.
When the user clicks on a particular dismissed message, any dismissed messages with a matching message string are removed from the dismissed collection.
This command execution refers to the message's 'Tag' property, since the MenuItem displays a truncated version of the message to the user for legibility reasons, whereas the 'Tag' contains the original, unedited error message."